docs/tutorial_advanced.rst

```diff
@@ -200,7 +187,7 @@ Finally, we may even use a custom Makefile by setting the :attr:`Make <reframe.c
 
 
 As a final note, as with the :class:`SingleSource` build system, it wouldn't have been necessary to specify one in this test, if we wouldn't have to set the CPPFLAGS.
-ReFrame could automatically figure out the correct build system if :attr:`sourcepath <reframe.core.pipeline.RegressionTest.sourcepath>` refers to a directory.
+ReFrame could automatically figure out the correct build system if :attr:`~reframe.core.pipeline.RegressionTest.sourcepath` refers to a directory.
```

Comment:
I just found out about the trick of using `~` to get only the last component as the link name.